### PR TITLE
Use a single stream from Horizon, instead of a per-withdrawal.

### LINKED ISSUE #114: Have a single stream for Horizon transactions
Currently, in each withdrawal, a new stream of transactions from Horizon is created. Instead of doing this, have a single stream of Horizon transactions involving the anchor's Stellar account.

### PR DESCRIPTION
**Pull request by debnil**
This PR converts streaming from Horizon to use a single stream of Horizon responses, administered through a custom Django admin command, rather than the per-withdrawal Celery task we have been using. This closes #114.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,6 +5,16 @@ services:
     image: "redis:alpine"
     ports:
       - "6379:6379"
+  web:
```
Comment (msfeldstein) [inline docker-compose.yml]: What does it mean to have 2 targets named web

File: src/transaction/management/commands/watch_transactions.py [hunk 2]
```diff
@@ -86,17 +72,39 @@ def process_withdrawal(response, withdraw_memo):
...
-    app.worker_main()
+class Command(BaseCommand):
+    """
+    Custom command to monitor Stellar transactions to the anchor account.
+    It also updates the transactions in the database.
+    """
+
+    help = "Watches for Stellar transactions to the anchor account using Horizon"
+
+    def handle(self, *args, **options):
```
Comment (msfeldstein) [inline src/transaction/management/commands/watch_transactions.py]: Does this get called on every streaming stellar transaction?

File: docker-compose.yml (hunk 1 above)
Comment (debnil) [reply docker-compose.yml]: It means the logs are confusing, so I've changed one target to `web_2`.

File: src/transaction/management/commands/watch_transactions.py (hunk 2 above)
Comment (debnil) [reply src/transaction/management/commands/watch_transactions.py]: No, it just gets called once at start-up [here](https://github.com/stellar/stellar-anchor-server/pull/115/files#diff-4e5e90c6228fd48698d074241c2ba760R26).

File: src/transaction/management/commands/watch_transactions.py [hunk 3]
```diff
@@ -1,24 +1,16 @@
...
-@app.task
-def watch_stellar_withdraw(withdraw_memo):
-    """Watch for the withdrawal transaction over the Stellar network."""
-    transaction_data = get_transactions()
-    for transaction in transaction_data:
-        if process_withdrawal(transaction, withdraw_memo):
-            break
-
-
 def get_transactions():
```
Comment (msfeldstein) [inline src/transaction/management/commands/watch_transactions.py]: Could you rename this transactions_stream or something to be more clear that this is streaming and not just getting a batch?

File: src/transaction/management/commands/watch_transactions.py [hunk 4]
```diff
@@ -98,5 +84,23 @@ def process_withdrawal(response, withdraw_memo):
...
+    """
+
+    help = "Watches for Stellar transactions to the anchor account using Horizon"
+
+    def handle(self, *args, **options):
+        try:
+            transaction_responses = get_transactions()
+        except HorizonError as exc:
+            raise CommandError(exc.message)
+        for response in transaction_responses:
```
Comment (msfeldstein) [inline src/transaction/management/commands/watch_transactions.py]: I may just not understand python but does this for loop use some async magic to continue iterating over new transactions in the stream?

**Pull request merged**